src/Instagram.php

```diff
@@ -92,14 +92,35 @@ public function fetchMedia(AccessToken $token, $limit = 20)
             }
         }
 
-        return collect($response['data'])
-            ->map(function($media) {
+        $page = true;
+        $collection = collect($response['data']);
+        $page_count = 1;
+        while ($page !== false) {
+            if (isset($response['paging'])) {
+                if (isset($response['paging']['next'])) {
+                    $page = $response['paging']['next'];
+                    var_dump(['page'.$page_count]);
```

Comment:
Can we remove the var_dump?

Reply:
👍 Removed, thanks! I've also removed the `$page_count` variable

Reply:
Thanks, that's wonderful.